functions.php

```diff
@@ -748,6 +748,45 @@ public function register_header_metabox() {
 			],
 			'preview_size' => 'large',
 		] );
+
+		$p4_open_graph = new_cmb2_box( [
+			'id'           => $prefix . 'metabox_og',
+			'title'        => __( 'Open Graph/Social Fields', 'planet4-master-theme-backend' ),
+			'object_types' => [ 'page', 'post' ],
+			'closed'       => true,  // Keep the metabox closed by default.
+		] );
+
+		$p4_open_graph->add_field( [
+			'name' => __( 'Title', 'planet4-master-theme-backend' ),
+			'desc' => __( 'Enter title if you want to override the open graph title', 'planet4-master-theme-backend' ),
+			'id'   => $prefix . 'og_title',
+			'type' => 'text_medium',
+		] );
+
+		$p4_open_graph->add_field( [
+			'name'    => __( 'Description', 'planet4-master-theme-backend' ),
+			'desc'    => __( 'Enter description if you want to override the open graph description', 'planet4-master-theme-backend' ),
+			'id'      => $prefix . 'og_description',
+			'type'    => 'wysiwyg',
+			'options' => [
+				'media_buttons' => false,
+				'textarea_rows' => 5,
+			],
+		] );
+
+		$p4_open_graph->add_field( [
+			'name'         => __( 'Image Override', 'planet4-master-theme-backend' ),
+			'desc'         => __( 'Upload an image or select one from the media library to override the open graph image', 'planet4-master-theme-backend' ),
+			'id'           => $prefix . 'og_image',
+			'type'         => 'file',
+			'options'      => [
+				'url' => false,
+			],
+			'text'         => [
+				'add_upload_file_text' => __( 'Add Image', 'planet4-master-theme-backend' ),
+			],
+			'preview_size' => 'large',
```

Comment:
We can add `query_args`, so that the image selection popup will only list image options.
```
'query_args'   => array(
	'type' => 'image',
),
```